rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMService.java

```diff
@@ -118,25 +118,6 @@ String defineNodeSource(String sessionId, String nodeSourceName, String infrastr
             String[] policyParameters, String[] policyFileParameters, String nodesRecoverable)
             throws RestServerException, ServiceException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)